docs/whirlpool/docs/02-Architecture Overview/06-Whirlpool Parameters.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: Whirlpools Parameters
+---
+
+# Orca Whirlpool Parameters
+Orca Whirlpools uses the Whirlpools smart-contract to host liquidity for token pairs. Below are the set of constants Orca used to host their program.
+
+## Whirlpool Program ID
+| Cluster       | Address                                          |
+|---------------|--------------------------------------------------|
+| Mainnet-Beta  | whirLbMiicVdio4qvUfM5KAg6Ct8VwpYzGff3uctyCc      |
+| Devnet        | whirLbMiicVdio4qvUfM5KAg6Ct8VwpYzGff3uctyCc      |
+
+## WhirlpoolConfig Address
+| Cluster       | Address                                         |
+|---------------|-------------------------------------------------|
+| Mainnet-Beta  | 2LecshUwdy9xi7meFgHtFJQNSKk4KdTrcpvaB56dP2NQ    |
+| Devnet        | FcrweFY1G9HJAHG5inkGB6pKg1HZ6x9UC2WioAfWrGkR    |
+
+## WhirlpoolConfigExtension Address
+| Cluster       | Address                                         |
+|---------------|-------------------------------------------------|
+| Mainnet-Beta  | 777H5H3Tp9U11uRVRzFwM8BinfiakbaLT8vQpeuhvEiH    |
+| Devnet        | 475EJ7JqnRpVLoFVzp2ruEYvWWMCf6Z8KMWRujtXXNSU    |
+
+## Initialized FeeTier and TickSpacing
+
+### Trading Fees
+Pools with a fee tier of  ≥0.3% have the trading fee divided as follows: 
+- 87% to the maker (Liquidity Provider)
+- 12% to the DAO treasury
+- 1% to the Climate Fund
+
+For pools with a fee tier of lower than 0.3% all fees are paid to the maker (Liquidity provider).
+
+**Fee Rate**: When creating a new pool, users can select the fee rate paid by traders - a fee rate on a CLAMM is the same as a spread on an order-book.
```

Comment:
Is the fee-rate the same as the spread?

Reply:
Looked it up and doesn't seem to be true. Good spot.